src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, Area {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public void draw() {
+    }

```

Comment:
The `draw()` method in `Figure` class should not be empty. Since `Figure` is an abstract class, you can make `draw()` an abstract method and implement it in the subclasses.